src/main/scala/uk/gov/homeoffice/drt/analytics/PaxDeltas.scala

```diff
@@ -0,0 +1,30 @@
+package uk.gov.homeoffice.drt.analytics
+
+import uk.gov.homeoffice.drt.analytics.time.SDate
+
+object PaxDeltas {
+  def maybeAverageDelta(maybeDeltas: Seq[Option[Int]]): Option[Int] = {
+    val total = maybeDeltas.collect { case Some(diff) => diff }.sum.toDouble
+    maybeDeltas.count(_.isDefined) match {
+      case 0 => None
+      case daysWithNumbers => Option((total / daysWithNumbers).round.toInt)
+    }
+  }
+
+  def maybeDeltas(dailyPaxNosByDay: Map[(Long, Long), Int],
```

Comment:
dailyPaxNosByDay could have it's own type maybe? It's not clear which long is which in this tuple.

Reply:
I mean the key really, the map is fine as a data structure.